fern/definition/medical/document.yml

```diff
@@ -135,6 +141,9 @@ types:
     properties:
       download: optional<Progress>
       convert: optional<Progress>
+      requestId:
```

Comment:
so this requestId in the docs doesnt have a required flag next to it. But ofc its always returned by our endpoint. We arent really consistent about that. Some endpoints have required in the response next to values that are always returned and others dont. Maybe not a big deal but wanted to note